### PR TITLE
Update build document to reflect name change in fork

### PR DESCRIPTION
**Pull request by gnuchu**
Resolves: Documentation change only. 

Update docs to refer to Tenacity rather than Audacity.

- [x] The code in this pull request is licensed under "GPLv2 or any later version"
- [ ] I made sure the code compiles on my machine
- [ ] I made sure there are no unnecessary changes in the code
- [ ] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -80,71 +80,71 @@ $ sudo apt-get install libgtk2.0-dev libasound2-dev libavformat-dev libjack-jack
...
 Steps 1 and 2 are only required for first-time builds. 
 
-Alternatively, you can use **CLion**. If you chose to do so, open the directory where you have cloned Audacity using CLion and you are good to go.
+Alternatively, you can use **CLion**. If you chose to do so, open the directory where you have cloned Tenacity using CLion and you are good to go.
 
 At the moment we only support **x86_64** builds. It is possible to build using AppleSilicon hardware but **mad** and **id3tag** should be disabled:
 
 ```
-cmake -GXCode -T buildsystem=1 -Daudacity_use_mad="off" -Daudacity_use_id3tag=off ../audacity
+cmake -GXCode -T buildsystem=1 -Dtenacity_use_mad="off" -Dtenacity_use_id3tag=off ../tenacity
```
Comment (ajayyy) [inline BUILDING.md]: This cannot be changed yet

**Review by Semisol — DISMISSED** [review]
Just in case: I don't think arguments/names for commands are updated yet, so could you revert?

**Comment by gnuchu** [comment]
> Just in case: I don't think arguments/names for commands are updated yet, so could you revert?

Ah - I tried to avoid doing that. I'll change now and re-submit PR

File: BUILDING.md (hunk 1 above)
Comment (Be-ing) [reply BUILDING.md]: Issue regarding this: #127

**Comment by Be-ing** [comment]
> re-submit PR

Please don't scatter discussion by opening a new PR. Either add a new commit to this branch or use `git commit --amend` and force push.

**Review by Be-ing — APPROVED** [review]

**Review by falkTX — APPROVED** [review]
Looks fine. Though I still think we need to change the way `../tenacity` cmake root dir is handled.
Can be in another PR

File: BUILDING.md [hunk 2]
```diff
@@ -80,71 +80,71 @@ $ sudo apt-get install libgtk2.0-dev libasound2-dev libavformat-dev libjack-jack
...
-    $ git clone https://github.com/cookiengineer/audacity/
+    $ git clone https://github.com/tenacityteam/tenacity/
     ```
 
-2. Configure Audacity using CMake:
+2. Configure Tenacity using CMake:
    ```
    $ mkdir build && cd build
-   $ cmake -GXcode -T buildsystem=1 ../audacity
+   $ cmake -GXcode -T buildsystem=1 ../tenacity
```
Comment (ajayyy) [inline BUILDING.md:100]: Can these be changed?

Comment (gnuchu) [reply BUILDING.md:100]: To what? We've cloned to a directory called "tenacity"

**Pull request merged**